Library/OcMainLib/OpenCoreVault.c

```diff
@@ -44,6 +41,19 @@ OcGetVaultKey (
   UINT32   Index;
   BOOLEAN  AllZero;
 
+  STATIC_ASSERT (
+    sizeof (OC_RSA_PUBLIC_KEY_2048) == sizeof (((OC_RSA_PUBLIC_KEY_2048 *)NULL)->Hdr) + sizeof (((OC_RSA_PUBLIC_KEY_2048 *)NULL)->Data),
+    "OC_RSA_PUBLIC_KEY_2048 is not naturally packed"
+    );
```

Comment:
Hmmm, maybe I should add another(?) assert that the size of this is 528, as expected by the script.

Reply:
Okay, actually just added this.

Reply:
Actually that should be the only ASSERT. Yours is more “obvious” why that size is chosen, but less readable. :( I think we use only the size constant everywhere.

Reply:
Which one are you saying I should keep? (Btw I added the one for 528 because the script uses 528 hard coded - expand down a bit from where I edited the script in the PR.)

Reply:
528

Reply:
Yes, sgtm.